frontend/src/components/BottomNav.tsx

```diff
@@ -51,6 +53,8 @@ function BottomNav(props: Props) {
                                     dispatch({type: ActionTypes.changeStoreOpt, isSingleStore: false});
                                     // call loading screen
                                     dispatch({type: GlobalStates.loadingScreen})
+                                    // call to store list in local storage
+                                    storeItems(props.items);
```

Comment:
I know this isn't acutally part of your code, but this function and the one above it are really similar, so I think it might be better to extract the functionality to a common function like `handleDialogPress(isSingleStore: boolean)`

Reply:
Ah, I can go ahead and change that. Just make sure to use `git checkout frontend frontend/src/components/BottomNav.tsx` to bring it over to the local-storage branch.